tests/api_tests/users/test_views.py

```diff
@@ -191,3 +191,145 @@ def test_get_projects_logged_in_as_different_user(self):
         assert_not_in(self.private_project_user_two._id, ids)
         assert_not_in(self.folder._id, ids)
         assert_not_in(self.deleted_project_user_one._id, ids)
+
+
+class TestUserRoutesNodeRoutes(ApiTestCase):
+
+    def setUp(self):
+        ApiTestCase.setUp(self)
+        self.user_one = UserFactory.build()
+        self.user_one.set_password('justapoorboy')
+        self.user_one.social['twitter'] = 'howtopizza'
+        self.user_one.save()
+        self.auth_one = (self.user_one.username, 'justapoorboy')
+        self.user_two = UserFactory.build()
+        self.user_two.set_password('justapoorboy')
+        self.user_two.save()
+        self.auth_two = (self.user_two.username, 'justapoorboy')
+        self.public_project_user_one = ProjectFactory(title="Public Project User One", is_public=True, creator=self.user_one)
+        self.private_project_user_one = ProjectFactory(title="Private Project User One", is_public=False, creator=self.user_one)
+        self.public_project_user_two = ProjectFactory(title="Public Project User Two", is_public=True, creator=self.user_two)
+        self.private_project_user_two = ProjectFactory(title="Private Project User Two", is_public=False, creator=self.user_two)
+        self.deleted_project_user_one = FolderFactory(title="Deleted Project User One", is_public=False, creator=self.user_one, is_deleted=True)
+        self.folder = FolderFactory()
+        self.deleted_folder = FolderFactory(title="Deleted Folder User One", is_public=False, creator=self.user_one, is_deleted=True)
+        self.dashboard = DashboardFactory()
+
+    def tearDown(self):
+        ApiTestCase.tearDown(self)
+        Node.remove()
+
+    def test_path_Users_User_id_Nodes_user_not_logged_in(self): #~WORK
```

Comment:
Function names, including class methods, should not contain capitals.

ref: http://cosdev.readthedocs.org/en/latest/style_guides/python.html#naming